WordPress/src/main/java/org/wordpress/android/ui/stats/StatsVisitorsAndViewsFragment.java

```diff
@@ -404,6 +404,16 @@ private void updateUI() {
             secondarySeriesItems = new GraphView.GraphViewData[dataToShowOnGraph.length];
         }
 
+        // index of days that should be XXX on the graph
+        final boolean[] weekendDays;
+        switch (getTimeframe()) {
```

Comment:
This can be simplified to an `if/else` block, or ternary operator if you're feeling crazy :stuck_out_tongue_winking_eye: